app/views/documents/_inline_attachment.html.erb

```diff
@@ -1,4 +1,4 @@
-<%= content_tag_for(:span, attachment.becomes(Attachment), class: "inline") do %>
```

Comment:
Was the removal of `attachment.becomes(Attachment)` intentional?

Looks like it was originally added here:
https://github.com/alphagov/whitehall/commit/16e86759cd73d5422bc621b2079c29523e0ae735
(no reason why is included)